src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -367,6 +453,20 @@ class CreatePipelineRun extends React.Component {
               />
             </FormGroup>
           )}
+          <FormGroup legendText="">
+            <KeyValueList
+              legendText="Labels"
+              invalidText="Labels must follow the kubernetes labels syntax"
```

Comment:
Should we provide a link to documentation here or are users expected to know the rules?

Reply:
I can add a link to https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set. Since I'm passing this as the prop `invalidText`, and this string is embedded in `<p>{invalidText}</p>`, I don't know how to embed a link in the text...
Will I have to pass `invalidText={<p>...<a href="">...</a></p>}` and then refer to that within the KeyValueList as `{invalidText}`?

Reply:
I've updated the text to have a link 👍